.env.sample

```diff
@@ -96,15 +96,16 @@ POD_AGENT_HOST_KUBECONFIG_PATH=/home/wojtek/.kube
 POD_AGENT_METADATA_REMOTE_WRITE_URL=http://magpie-monitor-cluster-metadata-service:9090/v1/metadata/clusters
 POD_AGENT_METADATA_SCRAPE_INTERVAL_SECONDS=5
 POD_AGENT_LOG_SCRAPE_INTERVAL_SECONDS=5
-
+MAX_POD_PACKET_SIZE_BYTES=5000
+MAX_CONTAINER_PACKET_SIZE_BYTES=1000
```

Comment:
Consider prefixing it with AGENT or something similiar to make it clear which service uses it

Reply:
Good point, added AGENT prefix